internal/armada/repository/event_test.go

```diff
@@ -4,115 +4,222 @@ import (
 	"testing"
 	"time"
 
-	"github.com/G-Research/armada/internal/common/compress"
```

Comment:
Ignore the diff here- this is a new file.  The previous file has been moved to `legacy_event_test.go`